src/_guides/language/effective-dart/index.md

```diff
@@ -115,19 +110,23 @@ readable and maintainable code.
 To keep the guidelines brief, we use a few shorthand terms to refer to different
 Dart constructs.
 
-* A **member** is any definition or declaration in a library. This includes
-  top-level variables, getters, setters, and functions. It also includes
-  instance and static members inside classes: fields, getters, setters, methods,
-  and operators.
+* A **library member** is a top-level field, getter, setter or function.
```

Comment:
setter or -> setter, or

(Google is a serial-comma shop, but I was going to let non-serial commas slide until I saw that the next paragraph uses one in a very similar place!)

Reply:
Do,ne.